app.py

```diff
@@ -0,0 +1,52 @@
+from flask import Flask, redirect, render_template, request
+from flask_cors import CORS 
+from flask_sqlalchemy import SQLAlchemy
+from controllers import urls as Urls
+
+
+#from werkzeug import exceptions
+
+app = Flask(__name__)
+CORS(app)
+
+POSTGRES = {
+    'user': 'debmon',
+    'pw': 'secretpassword',
+    'db': 'shortererl',
+    'host': 'localhost',
+    'port': '5430',
+}
+app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql+psycopg2://%(user)s:%(pw)s@%(host)s:%(port)s/%(db)s' % POSTGRES
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+
+db = SQLAlchemy(app)
+
+@app.route('/', methods=['GET', 'POST'])
+def home():
+    if request.method == 'POST':
+        resp, code = Urls.create(request.form['original_url'])
+        return render_template('index.html', url=resp), code
+    else: 
+        return render_template('index.html')
+
+@app.route('/<short_url>')
```

Comment:
At the moment this takes me to this page 
![image](https://user-images.githubusercontent.com/74493318/131680439-759e268e-dfd0-47f7-a1c8-2c4004499afe.png)
and not the site, is it always supposed to do this? 
Also if I then click the link, it takes me to http://127.0.0.1:5000/www.github.com and not the github website